README.md

```diff
@@ -100,3 +100,25 @@ ETL::extract($extractor)
     ->transform($transformer)
     ->load($loader);
 ```
+
+## Run Tests
+
+Using the following command to do PHPUnit test:
+
+```bash
+composer test
+```
+
+By default, using the [pcov](https://pecl.php.net/package/pcov) to do code coverage work.
+
+If developers want to go back to use the PHPUnit code coverage, it can add following setting inside the `phpunit.xml` file:
```

Comment:
Could you please change it into something like: 

```
It's recommended to use [pcov](https://pecl.php.net/package/pcov) for code coverage however you can also use 
xdebug by setting `XDEBUG_MODE=coverage` env variable?
```

In general I don't want people to change the phpunit.xml file and it would be better if the CI and developers would use the same coverage tool (pcov in this case).